store/src/java/com/zimbra/cs/service/mail/ToXML.java

```diff
@@ -2082,14 +2085,19 @@ public static Element encodeMessageSummary(Element parent, ItemIdFormatter ifmt,
         if (!needToOutput(fields, Change.CONTENT)) {
             return el;
         }
-        boolean addRecips = (output == OutputParticipants.PUT_RECIPIENTS || output == OutputParticipants.PUT_BOTH);
-        boolean addSenders = (output == OutputParticipants.PUT_BOTH || output == OutputParticipants.PUT_SENDERS);
+        boolean addRecips = (output == OutputParticipants.PUT_RECIPIENTS || output == OutputParticipants.PUT_BOTH || output == OutputParticipants.PUT_ALL);
+        boolean addSenders = (output == OutputParticipants.PUT_BOTH || output == OutputParticipants.PUT_SENDERS || output == OutputParticipants.PUT_ALL);
+        boolean addAll = output == OutputParticipants.PUT_ALL;
```

Comment:
Better to put **output == OutputParticipants.PUT_ALL** in braces so that code is more readable.

Reply:
Done. Thanks!